docs/_posts/Meryem1425/2022-03-04-clinical_deidentification_glove_en_3_0.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: model
+title: Clinical Deidentification (glove)
+author: John Snow Labs
+name: clinical_deidentification_glove
+date: 2022-03-04
+tags: [deidentification, en, licensed, pipeline]
+task: Pipeline Healthcare
+language: en
+edition: Spark NLP for Healthcare 3.4.1
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This pipeline is trained with lightweight glove_100d embeddings and can be used to deidentify PHI information from medical texts. The PHI information will be masked and obfuscated in the resulting text. The pipeline can mask and obfuscate `AGE`, `CONTACT`, `DATE`, `ID`, `LOCATION`, `NAME`, `PROFESSION`, `CITY`, `COUNTRY`, `DOCTOR`, `HOSPITAL`, `IDNUM`, `MEDICALRECORD`, `ORGANIZATION`, `PATIENT`, `PHONE`, `PROFESSION`, `STREET`, `USERNAME`, `ZIP`, `ACCOUNT`, `LICENSE`, `VIN`, `SSN`, `DLN`, `PLATE`, `IPADDR` entities.
```

Comment:
- use `glove_100d`
- add `EMAIL` and please check with the other deid pipeline, both of them should be the same entities